### ISSUE TITLE
对3.6以下的版本不支持

### ISSUE THREAD

**Issue by elid0928** [qiniu/qmgo#309]
**Describe the bug**
A clear and concise description of what the bug is, and what you expected to happen.
在readme文档中看到支持的mongodb版本是2.6及以上， 但在测试时， 该并不支持mongodb 3.0.9, 根据项目介绍，是基于官方库go.mongodb.org/mongo-driver 的项目， 官方库标注的是仅支持mongodb 3.6 以上的版本

**To Reproduce**
Codes to reproduce the behavior.

driver: github.com/qiniu/qmgo v1.1.8
mongodb version: 3.0.9
go version:  1.20

server at 172.17.0.3:27017 reports wire version 3, but this version of the Go driver requires at least 6 (MongoDB 3.6)
new client fail server at 172.17.0.3:27017 reports wire version 3, but this version of the Go driver requires at least 6 (MongoDB 3.6)
connect error server at 172.17.0.3:27017 reports wire version 3, but this version of the Go driver requires at least 6 (MongoDB 3.6)

**Additional context**

**Comment by jiangz222** [comment]
Yes, I think official go driver  add this [limitation](https://github.com/mongodb/mongo-go-driver/pull/1063/files#diff-5219dd81fef81f689cbb0bc6e95580059a36af1d98731ae6239374b46c5788b7).
I can do Nothing but using early version.

**Comment by jiangz222** [comment]
I will update the `README`

**Comment by zandwang** [comment]
Looks like I'm going back to mgoV2
